packages/expo-cellular/ios/EXCellular/EXCellularModule.m

```diff
@@ -39,6 +39,22 @@ - (NSDictionary *)constantsToExport
   resolve(@([[self class] getCellularGeneration]));
 }
 
+
+EX_EXPORT_METHOD_AS(getCurrentCellularInfoAsync, getCurrentCarrierAsyncWithResolver:(EXPromiseResolveBlock)resolve rejecter:(EXPromiseRejectBlock)reject)
```

Comment:
Could you extract the logic to the separate function and also use it in `constantsToExport`?